cumulusci/core/config/ScratchOrgConfig.py

```diff
@@ -187,16 +187,17 @@ def create_org(self):
             "alias": sarge.shell_format(' -a "{0!s}"', self.sfdx_alias)
             if self.sfdx_alias
             else "",
-            "email": sarge.shell_format('adminEmail="{0!s}"', self.email_address)
+            "email": sarge.shell_format(' adminEmail="{0!s}"', self.email_address)
             if self.email_address and not org_def_has_email
             else "",
             "default": " -s" if self.default else "",
+            "instance": f" instance={self.instance}" if self.instance else "",
             "extraargs": os.environ.get("SFDX_ORG_CREATE_ARGS", ""),
         }
 
         # This feels a little dirty, but the use cases for extra args would mostly
         # work best with env vars
-        command = "force:org:create -f {config_file}{devhub}{namespaced}{days}{alias}{default}{wait} {email} {extraargs}".format(
+        command = "force:org:create -f {config_file}{devhub}{namespaced}{days}{alias}{default}{wait}{email}{instance}{extraargs}".format(
```

Comment:
I think we still need a space prior to `extraargs`.

```suggestion
        command = "force:org:create -f {config_file}{devhub}{namespaced}{days}{alias}{default}{wait}{email}{instance} {extraargs}".format(
```

Reply:
Good catch.